lib/protobuf/generators/field_generator.rb

```diff
@@ -20,7 +20,7 @@ class FieldGenerator < Base
       attr_reader :field_options
 
       def applicable_options
-        @applicable_options ||= field_options.map { |k, v| ":#{k} => #{v}" }
```

Comment:
My suggestion would be to explicitly use double quotes rather than relying on inspect since it's not totally obvious. Or maybe a comment to clarify?

Sent from my iPhone

> On Feb 25, 2016, at 12:44, Zach Margolis <notifications@github.com> wrote:
> 
> In lib/protobuf/generators/field_generator.rb:
> 
> > @@ -20,7 +20,7 @@ class FieldGenerator < Base
> >        attr_reader :field_options
> >  
> >        def applicable_options
> > -        @applicable_options ||= field_options.map { |k, v| ":#{k} => #{v}" }
> it's for extension names with a dot in the middle -- wouldn't be valid ruby syntax with :foo.bar so it has to be :"foo.bar"
> 
> —
> Reply to this email directly or view it on GitHub.
>